docs/docsite/rst/dev_guide/index.rst

```diff
@@ -15,17 +17,19 @@ Find the task that best describes what you want to do:
    * I want to :ref:`add a custom plugin or module locally <developing_locally>`.
    * I want to figure out if :ref:`developing a module is the right approach <module_dev_should_you>` for my use case.
 
-* I'm ready to start developing:
+* I've read the info above, and I’m sure I want to develop a module:
```

Comment:
Shippable is complaining about Unicode quotes in this line (instead of ASCII quotes). Possibly copy/pasted from a doc?